tests/gordo/client/test_forwarders.py

```diff
@@ -102,9 +103,10 @@ def test_influx_forwarder(influxdb, influxdb_uri, sensors, sensors_str):
     # Assign all keys unique numbers
     df = get_test_data(pd.MultiIndex.from_tuples(input_keys + output_keys))
 
+    client_machine = ClientMachine(**machine.to_dict())
     # Create the forwarder and forward the 'predictions' to influx.
     forwarder = ForwardPredictionsIntoInflux(destination_influx_uri=influxdb_uri)
-    forwarder.forward_predictions(predictions=df, machine=machine)
+    forwarder.forward_predictions(predictions=df, machine=client_machine)
 
     # Client to manually verify the points written
     client = influx_client_from_uri(influxdb_uri, dataframe_client=True)
```

Comment:
This whole  test could also be in `gordo-client` package

Reply:
I'l take a look on separate PR